docs/source/debugging.rst

```diff
@@ -0,0 +1,154 @@
+Common Errors and Warnings
+*********************
+
+A number of errors and warnings can be raised when attempting to solve a model.
+ 
+-``TypeError: unhashable type: ‘xxxx’`` - can be caused by passing something other than a constraint set as the second argument of a ``Model`` constructor
+ 
+-``TypeError: SignomialInequality could not simplify to a PosynomialInequality`` - this error occurs when there is a signomial constraint in a problem not solved with ``localsolve``. Either signomials must be enabled and ``localsolve`` used, or the signomial constraint must be turned into a GP constraint.
```

Comment:
error is now `InvalidGPConstraint("SignomialInequality could not simplify to a PosynomialInequality; try forming your program as SignomialProgram or calling .localsolve().")